.github/workflows/deploy.yml

```diff
@@ -0,0 +1,49 @@
+name: Deploy to GitHub Pages
```

Comment:
Do we really need this workflow file as we deploy on Netlify (and not on GitHub pages)?

Reply:
I just moved the files we had on docusaurus there sorry 😅 
I thought this was what was preventing me from merging this PR https://github.com/bump-sh/docs/pull/101 but now I'm not so sure.
Should I remove this file completely?